src/components/ChatInfo/TextBackground.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { FC } from "react";
+
+interface IProps {
+  width: number;
+  height: number;
+  deg: number;
+  left: number;
+  bottom: number;
+  right: number;
+  top: number;
+  zIndex: number;
```

Comment:
This prop should be optional, if you have assigned a default value for this prop. And then, you don't have to use prop zindex with value zero, because it's default value for this